README

```diff
@@ -0,0 +1,11 @@
+# azure-translator-python-sdk
```

Comment:
Why not keep the `.md` extension which allows proper parsing of the file on Github?